compiler/ir/backend.jvm/src/org/jetbrains/kotlin/backend/jvm/ir/IrUtils.kt

```diff
@@ -311,3 +312,21 @@ fun firstSuperMethodFromKotlin(
         owner.modality != Modality.ABSTRACT && owner.overrides(implementation)
     }
 }
+
+// MethodSignatureMapper uses the corresponding property of a function to determine correct names
+// for property accessors.
+fun IrSimpleFunction.copyCorrespondingPropertyFrom(source: IrSimpleFunction) {
+    val property = source.correspondingPropertySymbol?.owner ?: return
+    val target = this
+
+    correspondingPropertySymbol = buildProperty {
```

Comment:
Can we also copy the attributes from the property, as the previous code did, such as visibility, modality, modifiers, etc.?

Reply:
Done, since this is a bit cleaner. But note that the property in this case is only a placeholder to mark a function as a getter/setter. We're not necessarily linking up the getters/setters correctly.